language/stdlib/modules/block.mvir

```diff
@@ -0,0 +1,48 @@
+module Block {
+
+  resource T {
```

Comment:
What other fields will be added here?

Reply:
Time, the signatures on the last committed block, the block proposer, possibly a few more things.